### ISSUE TITLE
fromBlock for Staked events can start later

### ISSUE THREAD

**Issue by cygnusv** [nucypher/nuit#176]
There are 2 places where we read TokenStaking's `Staked` events:
https://github.com/nucypher/nuit/blob/dbd55fae4605b701964338f248c074f89333a60d/packages/react-app/src/pages/simplePRE/delegate.js#L26
https://github.com/nucypher/nuit/blob/dbd55fae4605b701964338f248c074f89333a60d/packages/react-app/src/pages/simplePRE/bondOperator.js#L26
In both places, instead of searching from block 0, we can start from `14113768` (for mainnet), which is the contract creation. Not sure if will speed things up or not, but it can't hurt!

**Comment by derekpierre** [comment]
It'll help somewhat - had similar experience with collecting events `nucypher status events`.

**Comment by vepkenez** [comment]
Yes, I originally had this but it's a pain when switching to different testnets.
Maybe I can default to 0 if it's not mainnet.
